src/controller/actions/topicDashboard.js

```diff
@@ -20,31 +20,35 @@ export const resourceError = error => ({
 export const ADD_RESOURCE = 'ADD_RESOURCE';
 export const addResource = resource => ({
   type: ADD_RESOURCE,
-  resource
+  payload: resource
 });
 
 export const UPDATE_RESOURCE = 'UPDATE_RESOURCE';
-export const updateResource = (resource, id) => ({
+export const updateResource = resource => ({
   type: UPDATE_RESOURCE,
-  resource,
-  id
+  payload: resource
 });
 
-export const DELETE_RESOURCE = 'DELETE_RESOURCES';
+export const UPDATE_RESC_ORDER = 'UPDATE_RESC_ORDER';
+export const updateRescourceOrder = rescOrder => ({
+  type: UPDATE_RESC_ORDER,
+  payload: rescOrder
+});
+export const DELETE_RESOURCE = 'DELETE_RESOURCE';
 export const delResource = id => ({
   type: DELETE_RESOURCE,
   id
 });
 
 /**
- *Gets all resources belonging to a parent
+ *Gets topic information for a specified topicID includeing resources, resource order, topic title, and ID
  *First dispatches a loading function to change state to loading
- *Second: Sends GET request to the server using api.resources
+ *Second: Sends GET request to the server using api.topic
  * Third: Dispatches resource_success action to add new data from server to current state
  * If there is an error, it dispatches an error obj to state
  * *@param {{id:integer}}} id
  */
-export const getResources = id => dispatch => {
+export const initializeTopicDasbhoard = id => dispatch => {
```

Comment:
Please fix this typo here and all places it's referenced.

```js
-  initializeTopicDasbhoard
+  initializeTopicDashboard
```
```

Reply:
Fixed :)